src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -122,6 +122,11 @@ private static void setupWindows(Application application) {
         CommandManagerWindowCommandChangeObserver windowObserver = new CommandManagerWindowCommandChangeObserver(
                 commandManager);
         CommandsFeature.getCommandManager().getChangePublisher().addSubscriber(windowObserver);
+
+        CommandsFeature.getCommandManager().getChangePublisher().addSubscriber(() -> {
+            commandManager.getCommandHistoryField().setText(commandManager.getCommandHistoryField().getText() + CommandsFeature.getCommandManager().getCurrentCommandString() + "\n");
```

Comment:
The goal of the history is to be able to reload commands used earlier by the application. Displaying its name is not enough.
It would be helpful also to see the loading time of commands.